### ISSUE TITLE
Support for variable length of push token

### ISSUE THREAD

**Issue by Tayfun74** [mac-cain13/notificato#64]
Hi,

Using Notificato we received some push tokens that were not a hexadecimal with a length of 64. In the APNs documentation it says:
```
IMPORTANT
APNs device tokens are of variable length. Do not hard-code their size.
```

Is there a specific reason to check on the length. Can you remove this check otherwise?
```
// Check if the length of the devicetoken is correct
if (64 != strlen($deviceToken)) {
     throw new \InvalidArgumentException('Invalid device token given, incorrect length: ' . $deviceToken . ' (' . strlen($deviceToken) . ')');
}
```

**Comment by mac-cain13** [comment]
Seems this has changed over time. Think this check should be removed, would you be able to submit a PR with this code removed and a link to the docs? Then I can merge it and release a new version.

**Comment by Tayfun74** [comment]
Yes can you give me the permission to push?

**Comment by mac-cain13** [comment]
You can fork this repo and then create a pull request from your fork to this one. See also: https://help.github.com/en/articles/creating-a-pull-request-from-a-fork

**Comment by Tayfun74** [comment]
Done https://github.com/mac-cain13/notificato/pull/65.

**Comment by mac-cain13** [comment]
Released your change in version 1.2.1
